lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -1,1767 +1,179 @@
-    }
-    .inner-container{
-      @include grid-container();
-      @include span(12);
-      padding: 0 1.25rem;
-    }
-    h2{
-      font-size: 1.5rem;
-      font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
-      margin: 1.25rem 0;
-      text-transform: none;
-    }
-    h1, h2, h3, h4, h5, h6{
-      font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
-    }
-    .col-centered{
-      float: none !important;
-      margin-left: auto !important;
-      margin-right: auto !important;
-    }
-    .input-group{
-      margin-bottom: 1.25rem;
-    }
-    .input-lg{
-      line-height: 55px !important;
-      height: 55px !important;
-      padding: 0 1.25rem !important;
-    }
-    .form-block{
-      label, input, select, button, textarea{
-        display: block;
-        width: 100%;
-        font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
-        font-style: normal;
-        margin-bottom: 0.5rem !important;
-        &.has-error{
-          border-color: palette(error, text);
-          border-radius: 4px 4px 0 0;
-          margin-bottom: 0 !important;
-          + .field-message{
-            border-radius: 0 0 4px 4px;
-            width: 100%;
-          }
-        }
-      }
-      input{
-        box-shadow: none;
-        border: 1px solid palette(grayscale, back);
-        border-radius: 4px;
-        font-size: 1rem;
-        width: 100%;
-        &:focus, &:active{
-          box-shadow: none;
-          border-color: palette(grayscale, base);
-        }
-      }
-      select{
-        -webkit-appearance: none;
-        -moz-appearance: none;
-        border-color: palette(grayscale, back);
-        appearance: none;
-        &::-ms-expand{
-          display: none;
-        }
-      }
-      textarea{
-        height: 280px;
-        border-color: palette(grayscale, back);
-        box-shadow: none;
-        resize: none;
-        padding: 1rem;
-      }
+    #program_video{
+      display:flex;
+      align-items: center;
+      height:100%;
       button{
-        margin-top: 2.5rem;
-        padding: 1.25rem;
-      }
-    }
-
-    .list-bulleted{
-      li{
-        margin-bottom: 0.2rem !important;
-        padding-left: 0.5rem !important;
-        line-height: 1.5rem;
-        color: palette(grayscale, base);
-        .brand-link{
-          display: inline;
-        }
+        background-color:rgba(0, 0, 0, 0)
       }
-    }
-
-    .btn-enroll-mobile{
-      margin-bottom: 2rem;
-      .btn-enroll{
-        max-width: 95%;
-        margin: auto;
-        border-radius: 3px;
+      iframe{
+        width: 100%;
+        max-width: 400px;
+        height: 100%;
+        max-height: 200px;
       }
-    }
-
-    #course-trailer {
-      .trailer-link {
-        color: $white;
-        font-weight: bold;
-
-        span {
-            text-decoration: underline;
-        }
-
-        &:hover {
-            cursor: pointer;
+      @media (max-width: 768px) {
```

Comment:
Use Bootstrap's helper functions that obey the configured breakpoints. Something like:

    @include media-breakpoint-up(md) {

You can read about these helpers here: https://getbootstrap.com/docs/4.0/layout/grid/

Reply:
I can't get those mixins to work with update_assets.
```
CompileError: Error: no mixin named media-breakpoint-down

       Backtrace:
       	lms/static/sass/views/_program-marketing-page.scss:60
        on line 60 of lms/static/sass/views/_program-marketing-page.scss
>>       @include media-breakpoint-down(md) {
```

Reply:
Is it possible that the partial is still included in a non-Bootstrap file? Those should work if Bootstrap is loaded.